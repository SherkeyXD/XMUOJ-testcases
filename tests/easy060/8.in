94
9
1 75 11 49 98 97 64 92 27 
2
5 14 
4
44 59 52 86 
10
4 52 91 74 17 63 27 29 73 44 
8
12 41 96 58 97 28 56 51 
9
92 4 43 64 47 93 11 23 63 
7
28 76 23 65 97 75 99 
9
9 79 49 10 27 1 54 59 26 
5
43 31 14 85 85 
10
14 48 14 6 35 63 78 51 75 45 
8
89 27 33 24 81 95 13 77 
10
18 52 17 67 93 2 6 96 99 25 
5
97 19 18 63 47 
4
73 61 7 71 
3
38 70 21 
2
15 18 
9
53 65 82 30 35 98 79 61 69 
8
22 82 69 40 75 45 83 41 
5
73 81 99 45 68 
10
8 97 79 48 34 67 29 95 96 74 
3
37 6 51 
9
60 4 20 89 99 22 98 92 27 
4
28 42 38 18 
7
30 90 71 12 64 69 76 
2
71 9 
9
38 92 72 81 35 34 96 62 51 
6
22 12 56 64 3 79 
3
17 15 74 
10
28 78 29 96 95 79 69 60 11 34 
7
80 78 71 97 80 81 75 
2
27 4 
8
5 34 79 35 50 35 23 7 
7
78 41 95 59 83 52 17 
6
76 65 80 87 61 56 
2
66 34 
9
61 45 1 67 65 32 16 64 24 
8
84 44 30 3 55 64 43 28 
7
72 39 96 50 68 34 48 
9
92 8 30 39 83 96 1 75 74 
8
59 81 52 39 6 18 31 22 
2
5 94 
9
86 25 55 26 62 58 59 50 52 
9
11 22 67 36 35 90 26 98 53 
3
85 7 89 
10
6 56 84 43 38 15 41 9 66 79 
3
84 38 75 
5
55 89 84 97 51 
3
61 38 26 
2
69 88 
3
44 66 27 
7
8 74 71 74 56 70 25 
9
87 32 95 35 27 76 9 49 39 
10
96 41 50 49 65 58 75 27 29 69 
4
31 32 5 29 
5
15 20 81 53 72 
9
53 50 45 26 64 28 93 2 63 
10
11 54 24 29 98 13 6 65 79 88 
3
67 74 52 
10
54 97 4 12 44 21 6 50 47 43 
10
18 96 20 58 7 56 4 79 27 30 
4
89 18 17 5 
7
68 41 35 52 28 92 62 
5
82 19 18 37 49 
10
83 25 92 45 71 32 99 1 97 22 
4
76 62 58 76 
5
31 72 82 12 9 
4
69 57 12 5 
6
48 19 35 10 61 21 
10
29 34 48 10 57 19 73 14 62 90 
7
61 28 46 40 58 14 38 
6
79 49 53 7 79 44 
7
91 78 67 40 70 90 95 
10
52 52 77 85 73 7 96 87 81 14 
7
31 77 28 1 75 4 25 
5
31 67 15 72 54 
9
17 50 19 1 48 5 52 1 86 
2
31 95 
3
68 84 17 
9
27 46 43 47 20 75 53 56 83 
2
73 45 
9
60 53 70 66 44 49 83 93 86 
8
19 28 23 3 13 66 94 52 
6
92 61 2 6 80 85 
3
62 71 76 
4
28 80 81 24 
2
11 11 
6
97 40 67 33 20 76 
9
12 39 13 25 38 77 89 96 74 
5
23 54 5 3 71 
8
42 40 3 77 19 78 57 61 
3
13 55 38 
5
39 29 70 2 94 
8
81 28 2 86 16 40 3 27 
4
91 10 62 41 
