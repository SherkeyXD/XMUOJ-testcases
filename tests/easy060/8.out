11
14
52
17
28
11
28
9
31
14
24
6
19
61
38
18
35
40
68
29
37
20
28
30
71
35
12
17
28
75
27
7
41
61
66
16
28
39
8
18
94
26
22
85
9
75
55
38
88
44
25
27
29
29
20
26
11
67
6
7
17
35
19
22
62
12
12
19
14
28
44
67
14
4
31
1
95
68
27
73
49
13
6
71
28
11
33
13
5
19
38
29
3
41
